lib/twilio-ruby/security/request_validator.rb

```diff
@@ -31,7 +30,7 @@ def secure_compare(a, b)
 
         res = 0
         b.each_byte { |byte| res |= byte ^ l.shift }
-        res == 0
+        res.zero?
```

Comment:
https://ruby-doc.org/core-2.0.0/Numeric.html#method-i-zero-3F

This is just a rubyism for "intentionality" or concision I believe, same as `.nil?`.